lib/right_api_client/client.rb

```diff
@@ -485,7 +485,11 @@ def do_put(path, params={})
 
       begin
         retry_request do
-          @rest_client[path].put(params, headers) do |response, request, result|
+          # Altering headers to set Content-Type to text/plain when updating rightscript content
+          put_headers =
```

Comment:
quick question, would this be clearer outside the begin rescue block?  unless you see a reason for it.  Is it because of the retry_request?  Just thinking it would be a better pattern to have the code that is likely to trigger the exception in the begin rescue block?